File: 3. Python/02. Patterns/basic_patterns.py
rows = 5

print("Right triangle pattern:")
for i in range(1, rows + 1):
    print("* " * i)

print("\nNumber pattern:")
for i in range(1, rows + 1):
    for j in range(1, i + 1):
        print(j, end=" ")
    print()

File: 1. C++/02. Patterns/basic_patterns.cpp
#include <iostream>
using namespace std;

int main() {
    int rows = 5;

    cout << "Right triangle pattern:\n";
    for (int i = 1; i <= rows; i++) {
        for (int j = 1; j <= i; j++) {
            cout << "* ";
        }
        cout << "\n";
    }

    cout << "\nNumber pattern:\n";
    for (int i = 1; i <= rows; i++) {
        for (int j = 1; j <= i; j++) {
            cout << j << ' ';
        }
        cout << "\n";
    }

    return 0;
}
